### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by selten**
Use code blocks where they seem appropriate, format the data manager portion of the documentation further. 

Sorry for not including this in the last pull request to update this documentation.

### CODE REVIEW

File: doc/source/dev/build_a_job_runner.rst [hunk 1]
```diff
@@ -45,8 +45,8 @@ framework and the external executor framework. To know, when and how
 these methods are invoked, we will see about the implementation of
 parent class and process lifecycle of the runner.
 
-Implementation of parent class (galaxy.jobs.runners.\_\_init\_\_.py)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Implementation of parent class (``galaxy.jobs.runners.__init__.py``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (nsoranzo) [inline doc/source/dev/build_a_job_runner.rst]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/source/dev/build_a_job_runner.rst [hunk 2]
```diff
@@ -64,28 +64,28 @@ purpose.
...
+1. ``__init__`` method - STAGE 1
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Input params:
 
-1. app
+1. ``app``
 
-2. nworkers (Number of threads specified in job\_conf)
+2. ``nworkers`` (Number of threads specified in ``job_conf``)
```
Comment (nsoranzo) [inline doc/source/dev/build_a_job_runner.rst]: Should that be `job_conf.xml` ?

File: doc/source/dev/build_a_job_runner.rst [hunk 3]
```diff
@@ -107,30 +107,30 @@ Have a look at the sample job\_conf.xml:
...
+.. code-block:: python
 
     super(GodockerJobRunner, self).__init__(app, nworkers, **kwargs)
 
 D: The assigned values can be accessed in runner in the following way.
 
-::
+.. code-block:: python
 
     print self.runner_params["user"]
```
Comment (nsoranzo) [inline doc/source/dev/build_a_job_runner.rst]:
```suggestion
    print(self.runner_params["user"])
```

File: doc/source/dev/build_a_job_runner.rst [hunk 4]
```diff
@@ -107,30 +107,30 @@ Have a look at the sample job\_conf.xml:
...
 
     super(GodockerJobRunner, self).__init__(app, nworkers, **kwargs)
 
 D: The assigned values can be accessed in runner in the following way.
 
-::
+.. code-block:: python
 
     print self.runner_params["user"] 
     print self.runner_params["key"]
```
Comment (nsoranzo) [inline doc/source/dev/build_a_job_runner.rst]:
```suggestion
    print(self.runner_params["key"])
```

File: doc/source/dev/data_managers.rst [hunk 5]
```diff
@@ -441,11 +446,12 @@ An example single entry data_manager_config_file
         </data_manager>
     </data_managers>
 
-An example data_manager/fetch_genome_all_fasta.xml
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-This Tool Config calls a python script data_manager_fetch_genome_all_fasta.py and provides a single file out_file and the description from the dbkey dropdown menu for input.
+An example ``data_manager/fetch_genome_all_fasta.xml``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This Tool Config calls a python script ``data_manager_fetch_genome_all_fasta.py`` and provides a single file ``out_file`` and the description from the dbkey dropdown menu for input.
```
Comment (nsoranzo) [inline doc/source/dev/data_managers.rst]:
```suggestion
This Tool Config calls a Python script ``data_manager_fetch_genome_all_fasta.py`` and provides a single file ``out_file`` and the description from the dbkey dropdown menu for input.
```

File: doc/source/dev/data_managers.rst [hunk 6]
```diff
@@ -385,21 +388,22 @@ Data Managers are composed of two components:
...
 
-The primary difference between a standard Galaxy Tool and a Data Manager Tool is that the primary output dataset of a Data Manager Tool must be a file containing a JSON description of the new entries to add to a Tool Data Table. The on-disk content to be referenced by the Data Manager Tool, if any, is stored within the extra_files_path of the output dataset created by the tool.
+The primary difference between a standard Galaxy Tool and a Data Manager Tool is that the primary output dataset of a Data Manager Tool must be a file containing a JSON description of the new entries to add to a Tool Data Table. The on-disk content to be referenced by the Data Manager Tool, if any, is stored within the ``extra_files_path`` of the output dataset created by the tool.
 
-A data manager tool can use a ``conda`` environment if the target Galaxy is version 18.09 or above (specified in tool.xml)/
+A data manager tool can use a ``conda`` environment if the target Galaxy is version 18.09 or above (specified in the tool's XML file).
```
Comment (nsoranzo) [inline doc/source/dev/data_managers.rst]:
```suggestion
A data manager tool can use a ``conda`` environment if the target Galaxy is version 18.09 or above (specified in the tool's XML file).
```

File: doc/source/dev/faq.rst [hunk 7]
```diff
@@ -7,14 +7,18 @@ to provide relatively concrete answers for specific Galaxy development scenarios
...
+
+.. code-block:: python
 
     python -i scripts/db_shell.py
 
 ... build Galaxy Javascript frontend client?
 --------------------------------------------
 
-We've added a makefile which will let you do this. You can simply run::
+We've added a makefile which will let you do this. You can simply run:
```
Comment (nsoranzo) [inline doc/source/dev/faq.rst]:
```suggestion
We have added a Makefile which will let you do this. You can simply run:
```

File: doc/source/dev/data_managers.rst [hunk 8]
```diff
@@ -4,23 +4,39 @@ Data managers
...
-Data Managers are a special class of Galaxy tool which allows for the download and/or creation of data that is stored within `Tool Data Tables <https://galaxyproject.org/admin/tools/data-tables/>`_ and their underlying flat (e.g. ``.loc``) files. These tools handle e.g. the creation of indexes and the addition of entries/lines to the data table / ``.loc`` file via the Galaxy admin interface.
+Data Managers are a special class of Galaxy tool which allows for the
+download and/or creation of data that is stored within
+`Tool Data Tables <https://galaxyproject.org/admin/tools/data-tables/>`_
+and their underlying flat (e.g. ``.loc``) files. These tools handle
+e.g. the creation of indexes and the addition of entries/lines to the
+data table / ``.loc`` file via the Galaxy admin interface.
 
-Data Managers can be defined locally or installed through the Tool Shed.
+Data Managers can be defined locally or installed through the ToolShed.
```
Comment (nsoranzo) [inline doc/source/dev/data_managers.rst]: I think the official spelling is "Tool Shed", although we have not been very consistent.

Comment (selten) [reply doc/source/dev/data_managers.rst]: I noticed this later, both are being used. I'll change to `Tool Shed`.

**Comment by dannon** [comment]
Thanks for this!  Test failure unrelated.

**Pull request merged**